meta-llama/Llama-3.1-8B-Instruct/storage.yml

```diff
@@ -0,0 +1,3 @@
+# storage config for Meta-Llama-3.1-8B-Instruct
+model: meta-llama/Meta-Llama-3.1-8B-Instruct
```

Comment:
For now, the hf model path is repeated in both the folder name and in this storage config (where it has to live). Do we want to make the folder name load bearing like the storage config, or is it just informational? If it's just informational then the side effect of making subfolders is a little odd to me

ie. if data is `gcp` instead of `hf`, would we still have the folder name be the hf_name or something else? Like if it was an internal checkpoint or code-named model.

Reply:
i'm choosing the folder name to match the stub to make it easy to find the "configs".